src/python_tools/BUILD

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@python_tools_deps//:requirements.bzl", "requirement")
+
+py_test(
+    name = "test_hermetic_python_environment",
+    srcs = ["test_hermetic_python_environment.py"],
+)
+
+py_binary(
+    name = "jupyter",
+    srcs = ["jupyter.py"],
+    deps = [
+        requirement("jupyter"),
+        requirement("notebook"),
+    ],
+)
```

Comment:
should `python_tools` be its own folder in the top level directory? Thoughts on having this under `//software/tools:jupyter_notebook`?

Reply:
my idea for this was that `software` would be C++ land and all the python data analysis stuff would go into `python_tools`